lib/phoenix/live_dashboard/layout_view.ex

```diff
@@ -27,7 +27,7 @@ defmodule Phoenix.LiveDashboard.LayoutView do
       Phoenix.VerifiedRoutes.unverified_path(
         conn,
         conn.private.phoenix_router,
-        "#{prefix}/#{asset}-#{hash}"
+        Path.join([prefix, "#{asset}-#{hash}"])
```

Comment:
This may use `\` on Windows. We would probably need to check the prefix explicitly. Perhaps we should rather normalize `__live_dashboard_prefix__` to return `""` as the prefix when mounted on root?